lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -316,7 +316,7 @@ defmodule ExUnit.Case do
   @doc """
   Defines a test with `message`.
 
-  The test may also define a `var`, which will pattern match
+  The test may also define a `var` or map, which will pattern match
```

Comment:
```suggestion
  The test may also define a pattern, which will be matched
```

Reply:
Feel like this is more correct

Reply:
If you want to specify a map will be returned, I'd write something like: 

... which will be matched against the context map